src/NuGet.Services.EndToEnd/AutoCompleteResultTests.cs

```diff
@@ -71,6 +74,9 @@ public AutocompleteResultTests(PushedPackagesFixture pushedPackages, ITestOutput
             // Arrange
             var package = await _pushedPackages.PrepareAsync(packageType, _logger);
 
+            // Wait for package to become available
```

Comment:
Can we also assert non-empty result of `v2Response` or `v3Response`?

Reply:
Good one. Added https://github.com/NuGet/NuGet.Services.EndToEnd/pull/23/commits/955cf7d2d6a2674d1a186afc53d83d3b44fb9a91